From f410871470773c0767f97f86c1bd05074db63081 Mon Sep 17 00:00:00 2001
From: "W.C.A. Wijngaards" <wouter@nlnetlabs.nl>
Date: Mon, 3 Feb 2020 10:37:26 +0100
Subject: [PATCH] - Fix for #3: Allow @ character to make scripts work, which
 may fix resolv.conf lost in some situation bug.

Changelog:
3 February 2020: Wouter
	- Fix for #3: Allow @ character to make scripts work, which may
	  fix resolv.conf lost in some situation bug.
---
 riggerd/ubhook.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/riggerd/ubhook.c b/riggerd/ubhook.c
index 382eee3..f1ce73c 100644
--- a/riggerd/ubhook.c
+++ b/riggerd/ubhook.c
@@ -80,7 +80,7 @@ allowed_arg(const char* arg)
 		}
 		if( isalnum((unsigned char)*s) || *s == ' ' || *s == ':' ||
 			*s == '.' || *s == '_' || *s == '-' || *s == '+' ||
-			*s == '\t') {
+			*s == '\t' || *s == '@') {
 			continue;
 		} else 	{
 			log_err("command line string argument '%s' fails check on allowed characters", arg);
-- 
2.41.0

